### ISSUE TITLE
Messages are not announced to screen reader

### ISSUE THREAD

**Issue by jrieken** [Microsoft/vscode#2531]
* open F1
* select Debug: Continue
* message appears that saying I am not debugging but explorer has focus, message isn't read

**Comment by bpasero** [comment]
@jrieken using voiceover or nvda?

**Comment by jrieken** [comment]
voiceover
